lib/bitcoin/README.md

```diff
@@ -66,14 +47,16 @@ Linux users:
 ifconfig
 ```
 
-If you installed Bcoin locally, `localhost` will do.
+If you installed Bitcoin locally, `localhost` will do.
 
-In the following configuration `bcoin.local` refers to your IP address, DNS address, or `localhost`.
+In the following configuration `bitcoin.local` refers to your IP address, DNS address, or `localhost` of the Bitcoin peer.
 
 Edit your bitcoin-config.json
 ```json
 {
-  "bitcoinPeerUri": "[FILL THIS IN!]",
+  "bitcoinPeerUri": "bitcoin.local:18332",
+  "bitcoinRpcUsername": "{{YOUR USERNAME}}",

```

Comment:
Nit: keep to the same convention, either use [FILL THIS IN!], or replace line 60 with something like {{YOUR WIF STRING}}

Reply:
Would you have a wallet at this point? Further on I mention that if you don't it'll complain and generate a WIF for you to use in this config value.